### PR TITLE
90-Bug-EnemyAction

### PR DESCRIPTION
**Pull request by MLHumphries**
-Fixed bug where enemy would not select another attack.

### CODE REVIEW

File: dunwoody_disaster/FightSequence.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def evaluateEnemyTurn():
 
         def finishTurn():
             playerActions.clear()
+            enemyActions.clear()
             enemyActions.selectRandom()
```
Comment (chanson02) [inline dunwoody_disaster/FightSequence.py:94]: Thank you Mr. bug hunter!
Maybe in `selectRandom` it should call `clear`?

**Review by chanson02 — APPROVED** [review]
Thanks for this fix.

File: dunwoody_disaster/FightSequence.py (hunk 1 above)
Comment (MLHumphries) [reply dunwoody_disaster/FightSequence.py:94]: Well someone was kind enough to point out where it was likely happening. S/O to that individual! 

I could make that change if needed.

**Review by instMeyFTW — APPROVED** [review]
nice!

**Pull request merged**